src/sql/sqlops.d.ts

```diff
@@ -347,6 +361,21 @@ declare module 'sqlops' {
 		 * The Operating System version string of the machine running the instance.
 		 */
 		osVersion: string;
+		/**
+		 * options for all new server properties.
+		 */
+		options: ServerInfoOption;
```

Comment:
As discussed last week this should be a more generic dictionary type that lets extensions add specific additional information, then the mssql extension can contribute `isBigDataCluster` and `clusterEndpoints` itself